src/Transition/CreateNewIDFUsingRulesV9_2_0.f90

```diff
@@ -480,6 +485,28 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with W, insert the rules here
 
               ! If your original object starts with Z, insert the rules here
+              CASE('ZONEHVAC:EQUIPMENTLIST')
+                nodiff = .false.
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                DO CurField = 1, CurArgs
+                  IF (CurField < 3) THEN
+                    zeqHeatingOrCooling = 'Neither'
+                  ELSE IF (MOD((CurField - 2) - 5, 6) == 0) THEN
+                    zeqHeatingOrCooling = 'Cooling'
+                  ELSE IF (MOD((CurField - 2) - 6, 6) == 0) THEN
+                    zeqHeatingOrCooling = 'Heating'
+                  ELSE
+                    zeqHeatingOrCooling = 'Neither'
+                  END IF
+                  IF (InArgs(CurField) /= Blank .AND. (zeqHeatingOrCooling == 'Cooling' .OR. zeqheatingOrCooling == 'Heating')) THEN
+                    zeqNum = (CurField - 3) / 6 + 1
+                    WRITE(zeqNumStr, *) zeqNum
+                    OutArgs(CurField) = TRIM(InArgs(1)) // ' ' // zeqHeatingOrCooling // 'Frac' // TRIM(ADJUSTL(zeqNumStr))
+                    CALL WriteOutIDFLines(DifLfn,'Schedule:Constant',3,[OutArgs(CurField), '', InArgs(CurField)],['Name                     ', 'Schedule Type Limits Name', 'Hourly Value             '],['', '', ''])
```

Comment:
This is well-crafted. I've mode a few modifications. Transitioned file comes out almost exactly the same. Added a schedule type object to keep the warnings down.

Reply:
Thanks for fixing my iffy transition code.